packages/sakuli-commons/package.json

```
@@ -28,20 +28,20 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/common-tags": "^1.8.0",
-    "@types/jest": "^24.0.13",
+    "@types/jest": "^25.1.4",
     "@types/node": "^11.13.5",
     "@types/properties-reader": "^0.0.1",
-    "@types/yargs": "^12.0.12",
     "jest": "^24.7.1",
```

Jest should be updated here as well.
```suggestion
    "jest": "^25.2.4",
```